### DISCUSSION TITLE
downloading packages from the SOC screen

### DISCUSSION THREAD

**Discussion by azxqw** [Security-Onion-Solutions/securityonion#2518] [Unsupported Versions] [answered]
Hi all, 
I have a problem downloading packages from the downloads screen in the SOC.
i get "Network Error" every time i try to download a package. Is there a rule within SO that i need to open in order to download the packages?

**Comment by jertel** [comment, marked as answer]
If you can access the SOC UI then there shouldn't be any rule preventing you from downloading packages from the SOC download screen. Do you get the same error for Wazuh agent downloads and osquery package downloads?

**Reply by azxqw** [reply]
it looks like we had temporary problems within our network that interrupted my downalods
it's all good now, so.. false alarm!
